index.js

```diff
@@ -26,6 +26,7 @@ export { default as PasswordValidationField } from './lib/PasswordValidationFiel
 export { default as ProxyManager } from './lib/ProxyManager';
 
 export { default as SearchAndSort } from './lib/SearchAndSort';
+export { default } from './lib/SearchAndSort/components/MultiSelectionFilter';
```

Comment:
Does this work? Shouldn't it be `export { default as MultiSelectionFilter } from './lib/...`?